Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/OperationMode.mo

```diff
@@ -38,12 +38,12 @@ block OperationMode "Block that outputs the operation mode"
     final quantity="Time") "Maximum warm-up time among all the zones"
     annotation (Placement(transformation(extent={{-300,130},{-260,170}}),
         iconTransformation(extent={{-140,40},{-100,80}})));
-  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occHeaHigMin
-    "True when the occupied heating setpoint temperature is higher than the minimum zone temperature"
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uOccHeaHig
+    "True when the occupied heating setpoint temperature is higher than the zone temperature"
     annotation (Placement(transformation(extent={{-300,90},{-260,130}}),
         iconTransformation(extent={{-140,20},{-100,60}})));
-  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput maxHigOccCoo
-    "True when the maximum zone temperature is higher than the occupied cooling setpoint"
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uHigOccCoo
+    "True when the zone temperature is higher than the occupied cooling setpoint"
```

Comment:
Similar as previously, may keep it as before. This would be the zone temperature for single zone system.

For the multi-zone system, the `false` singe here means that the maximum zone temperature is lower than the occupied cooling setpoint (so all zones temperature are low), the cool-down mode should be finished.

Reply:
See above.